localstack/services/cloudformation/models/ec2.py

```diff
@@ -331,6 +342,10 @@ def _pre_delete(resource_id, resources, resource_type, func, stack_name):
                     ]
                 )
                 for rt in resp["RouteTables"]:
+                    for assoc in rt["Associations"]:
```

Comment:
nit: Can we assume that `Associations` is always present? Let's better use `rt.get("Associations", [])` here to avoid `KeyError`.